Makefile

```diff
@@ -3,12 +3,11 @@
 all: build start
 
 build:
-	docker build --rm -t okibot/opentrials-airflow .
+	docker build --rm -t okibot/opentrials-airflow . -f Dockerfile
+	docker build --rm -t okibot/opentrials-airflow-worker . -f Dockerfile.worker
```

Comment:
Shouldn't we pass the `AIRFLOW_USER` env var here and nuke the worker Dockerfile?